students/ian_letourneau/Lesson08/test_circle_lab.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+# Ian Letourneau
+# 6/4/2018
+
+import math
+import circle_lab as cr
+
+# Test Step One
+
+
+def test_circle_creation():
+    c = cr.Circle(5)
+    assert c.radius == (5)
+
+    new_c = cr.Circle(23)
+    assert new_c.radius == (23)
+
+# Test Step Two
+
+
+def test_diamter_creation():
+    c = cr.Circle(5)
+    assert c.diameter == (10)
+
+    new_c = cr.Circle(23)
+    assert new_c.diameter == (46)
+
+# Test Step Three
+
+
+def test_diameter_setter():
+    c = cr.Circle(5)
+    assert c.diameter == (10)
+
+    c.radius = 15
+    assert c.diameter == (30)
+
+    c.diameter = 20
+    assert c.radius == (10)
+
+# Test Step Four
+
+
+def test_area():
+    c = cr.Circle(5)
+    assert c.area == ((math.pi*5)*2)
+
+    c.radius = 10
+    assert c.area == ((math.pi*10)*2)
+
+    c.diameter = 30
+    assert c.area == ((math.pi*15)*2)
+
+# Test Step Five
+
+
+def test_from_diameter():
+    c = cr.Circle.from_diameter(10)
+    assert c.diameter == (10)
+    assert c.radius == (5)
+
+# Test Step Six
+
+
+def test_str_repr():
+    c = cr.Circle(5)
+    print (c)
+    assert repr(c) == ("'Circle(5.0)'")
+
+    d = eval(repr(c))
+    assert d == ('Circle(5.0)')
+
+# Test Step Seven
+
+
+def test_numeric_protocol():
+    c1 = cr.Circle(2)
+    c2 = cr.Circle(4)
+
+    c3 = c1 + c2
+    assert eval(repr(c3)) == ('Circle(6.0)')
+
+    c4 = c2*3
+    assert eval(repr(c4)) == ('Circle(12.0)')
+
+    c5 = 3*c2
+    assert eval(repr(c5)) == ('Circle(12.0)')
+
+# Test Step Eight
+
+
+def test_comparison_overrides():
+    c1 = cr.Circle(2)
+    c2 = cr.Circle(4)
+
+    assert (c1 > c2) == False
+    assert (c1 < c2) == True
+    assert (c1 == c2) == False
```

Comment:
for true checks, you don't to test for True, that's what `assert` provides:
`assert c1 < c2` is sufficient

for False, you can either:
`assert not c1 > c2` or `assert c1 > c2 is False`

Using `==` with True/False isn't pythonic.